opensearch_dashboards.json

```diff
@@ -1,5 +1,5 @@
 {
-    "id": "customImportMap",
+    "id": "customImportMapDashboards",
```

Comment:
Is it safe to rename plugin id? cc: @ashwin-pc

Reply:
Actually, we are using customImportMapDashboards at released branches and 2.x, but just not in main branch. So I think we need keep consistent for released usage. cc @peterzhuamazon 

https://github.com/opensearch-project/dashboards-maps/blob/2.4/src/plugins/custom_import_map/opensearch_dashboards.json#L2

https://github.com/opensearch-project/dashboards-maps/blob/2.4/src/plugins/custom_import_map/opensearch_dashboards.json#L2